libs/state/src/lib/active-ftu/active-ftu.state.ts

```diff
@@ -37,7 +39,7 @@ export class ActiveFtuState {
    */
   @Action(Load, { cancelUncompleted: true })
   load({ getState, patchState, dispatch }: Context, { iri }: Load): Observable<void> | void {
-    if (getState().iri !== iri) {
+    if (iri && getState().iri !== iri) {
```

Comment:
`iri` cannot be undefined here. Are you testing for the empty string? If so why is it ever passing the empty string as the active iri?